tsl/test/expected/chunk_column_stats-14.out

```diff
@@ -51,21 +51,21 @@ INSERT INTO sample_table VALUES (:'start_date'::timestamptz, 16, 14.6554, 47, 'n
 -- Non-int, date, timestamp cannot be specified as a min/max ranges for now
```

Comment:
Does the test file name itself as `chunk_column_stats` seem alright? Would it need `chunk_skipping` in the name?

Reply:
To make it simple and fast I've decided to only rename the public API.